plugins/SitesManager/lang/en.json

```diff
@@ -85,8 +85,11 @@
         "SiteWithoutDataOtherIntegrations": "Other integrations",
         "SiteWithoutDataGoogleTagManager": "Google Tag Manager",
         "SiteWithoutDataMatomoTagManager": "Matomo Tag Manager",
+        "SiteWithoutDataSinglePageApplication": "Single-Page Application or Progressive Web App",
+        "SiteSinglePageOrProgressWebApplication": "Single-Page/Progressive Web",
```

Comment:
What about using `Single-Page or Progressive Web Application` for both? Imho we should try to use the same title in the link and as title. If it looks strange as the longer text might be split on multiple lines, we could try to prevent that using CSS.